docs/releases.md

```diff
@@ -35,3 +35,22 @@ Otherwise, we typically release every two weeks:
 
 * Patch fixes for the current stable version.
 * The next release candidate, if we are currently in a release-cycle.
+
+## Kubernetes Compatibility Matrix
+
+| Argo Workflows \ Kubernetes | 1.17 | 1.18 | 1.19 | 1.20 | 1.21 | 1.22 | 1.23 |
```

Comment:
1.24?

Reply:
1.25?

Reply:
https://github.com/argoproj/argo-workflows/commit/36646ef81cb4775c1ef31861f01331bc75166e7b

Reply:
@tooptoop4 @yolu-kxs did we support 1.24 and 1.25, I did not test on them?

Reply:
I am not sure if it supports 1.24 and 1.25. Has someone tested it?

Reply:
I have argo workflows 3.4.4 installed on k8s 1.24, it works as expected